backend/mo/ic_vetkeys/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change Log
+
+## [0.1.0] - 11-06-2025
```

Comment:
ISO 8601 style (YYYY-MM-DD) is IMO preferable as it is not ambiguous, since nobody (per https://en.wikipedia.org/wiki/Calendar_date#Date_format) uses YYYY-DD-MM. In North American convention 11-06-2025 is Nov 6.